src/core/types.ts

```diff
@@ -18,3 +18,37 @@ export type StreamSource = {
     options?: boolean | EventListenerOptions
   ): void
 }
+
+export type StructuredCloneValue =
```

Comment:
It would've been great if there was an existing TypeScript definition for compatible types. But I guess there isn't.

Reply:
I couldn't easily find one. I was hopeful on `Transferable`, but that's just `ArrayBuffer | ImageBitmap | MessagePort`, and unfortunately the VSCode `structuredClone` tooltips don't give any clues either 😞

Reply:
Yeah, it looks like TS's lib.dom defines the argument as `any`, which is really not helpful! :(